### DISCUSSION TITLE
Unable to download ZIP and tar files from GitHub

### DISCUSSION THREAD

**Discussion by MehulKhanna** [community/community#49389] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I am unable to download ZIP and tar.gz files from Github. These files are downloading at a speed of 10-30Kbps even with a 300Mbps Wifi. There is no problem with my wifi as I am able to download ZIPs and Tars from other sources without any issues. I have tested this on different devices as well.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @MehulKhanna ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
